File: solutions/python/raindrops/1/raindrops.py
def convert(number: int):
    raindrop = ""
    if not number % 3:
        raindrop += "Pling"
    if not number % 5:
        raindrop += "Plang"
    if not number % 7:
        raindrop += "Plong"
    return raindrop if raindrop else str(number)

File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"
#include <string>

namespace raindrops {

    std::string convert(int num){
        std::string result = "";
        if (num % 3 == 0){
            result = result + "Pling";
        }
        if (num % 5 == 0){
            result = result + "Plang";
        }
        if (num % 7 == 0){
            result = result + "Plong";
        }
        if (result.empty()){
            return std::to_string(num);
        }
        return result;
    }

}  // namespace raindrops
